syntax = "proto2";

message PandaLensMoments {
  repeated string moments = 1;
}